test/src/crypto.js

```diff
@@ -0,0 +1,149 @@
+import expect from 'expect.js';
+import sinon from 'sinon';
+import makeCryptoWith from '../../src/crypto';
```

Comment:
Hmmm... doing that screws up the code coverage report a bit - reading from `lib` reports a higher code coverage (presumably because there are more lines in the "compiled" code that are covered) and also screws up the uncovered line numbers (because they refer to the code in `lib` instead of `src`).